samples/tree.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Write your Calder code here
+ */
+
+const light1 = {
+    lightPosition: [10, 10, 10],
+    lightColor: [0.3, 0.3, 0.3],
+    lightIntensity: 256
+};
+const light2 = {
+    lightPosition: [700, 500, 50],
+    lightColor: [0.3, 0.3, 0.3],
+    lightIntensity: 100
+};
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Step 1: create geometry
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+// Add lights to the renderer
+renderer.addLight(light1);
```

Comment:
Nit: Should we move these two lines up ahead of the `Step 1` comment? They don't relate to creating the geometry, and we have the other light code above it.